### PR TITLE
Add new Apple Watch Series 4 sizes

### LINKED ISSUE #44: Icons for new Apple Watch Series 4
Maybe I can take it myself :)

### PR DESCRIPTION
**Pull request by robbiet480**
Hope no one minds that I added newlines to the Watch sizes, was getting too hard to read and modify!

Also fixes the broken tests!

Closes #44

### CODE REVIEW

**Comment by robbiet480** [comment]
@KrauseFx Sorry to directly ping you, I know you are constantly flooded with notifications, but I am sure that this is blocking some people from using the plugin, and as I said it also contains test fixes which would be good to get in. Hoping I can see this merged soon!

**Comment by robbiet480** [comment]
@KrauseFx Sorry to directly ping you, I know you are constantly flooded with notifications, but I am sure that this is blocking some people from using the plugin, and as I said it also contains test fixes which would be good to get in. Hoping I can see this merged soon!

**Comment by robbiet480** [comment]
@KrauseFx Sorry to directly ping you, I know you are constantly flooded with notifications, but I am sure that this is blocking some people from using the plugin, and as I said it also contains test fixes which would be good to get in. Hoping I can see this merged soon!

File: fastlane/Fastfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 lane :test do
   appicon(appicon_image_file: 'spec/fixtures/Themoji.png',
-    appicon_devices: [:ipad, :iphone, :ios_marketing, :watch, :watch_marketing])
```
Comment (KrauseFx) [inline fastlane/Fastfile:3]: Let's keep the basic `[` syntax here, to make it easier for non-Ruby devs 👍

Comment (KrauseFx) [inline fastlane/Fastfile:3]: Let's keep the basic `[` syntax here, to make it easier for non-Ruby devs 👍

File: fastlane/Fastfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    appicon_devices: %i[ipad iphone ios_marketing watch watch_marketing])
```
Comment (KrauseFx) [inline fastlane/Fastfile:3]: Let's keep the basic `[` syntax here, to make it easier for non-Ruby devs 👍

**Comment by mariohahn** [comment]
Any updates on the PR? 😏

**Comment by robbiet480** [comment]
Fixed via #47

**Pull request closed without merging**